### PR TITLE
Fix #177555 amomeupet.org

### LINKED ISSUE #177555: www.amomeupet.org
### Issue URL (Ads)
 [https://www.amomeupet.org/noticias/10860/apos-tratamento-mamae-gata-reencontra-filhote-e-cena-encanta-redes-sociais](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.amomeupet.org%2Fnoticias%2F10860%2Fapos-tratamento-mamae-gata-reencontra-filhote-e-cena-encanta-redes-sociais)
Username: `@arturstat `

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Android v4.3.1
System version: | Android 14
Browser: | Chrome
License type: | paid
AdGuard mode: | VPN
HTTPS filtering: | enabled
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Hide your Referrer from third-parties, <br>Hide your User-Agent, <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Self-destructing third-party cookies (180), <br>Self-destructing first-party cookies (4320), <br>Disable cache for third-party requests, <br>Block third-party Authorization header
DNS filtering: | server: `h3://dns.nextdns.io`<br>filters: `https://filters.adtidy.org/android/filters/15_optimized.txt`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection, <br/>Legitimate URL Shortener<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners<br/><br/><b>Security:</b><br/>Online Malicious URL Blocklist<br/><br/><details><summary><b>Language-specific:</b></summary>AdGuard Chinese, <br/>AdGuard Ukrainian, <br/>AdGuard German, <br/>AdGuard Dutch, <br/>AdGuard Russian, <br/>AdGuard French, <br/>AdGuard Japanese, <br/>AdGuard Turkish, <br/>AdGuard Spanish/Portuguese, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Bulgarian list, <br/>EasyList Hebrew, <br/>EasyList Italy, <br/>Adblock List for Finland, <br/>AdBlockID, <br/>Latvian List, <br/>EasyList Lithuania, <br/>Frellwit's Swedish Filter, <br/>Estonian List, <br/>Polish Anti Adblock Filters, <br/>IndianList, <br/>Macedonian adBlock Filters, <br/>EasyList Czech and Slovak, <br/>Hungarian, <br/>EasyList Thailand, <br/>Persian Blocker, <br/>ABPVN List, <br/>Liste AR, <br/>List-KR, <br/>Dandelion Sprout's Nordic Filters</details>
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js<br>https://userscripts.adtidy.org/release/disable-amp/1.0/disable-amp.meta.js

### PR DESCRIPTION
**Pull request by ksaduakasov**
Fix #177555

### CODE REVIEW

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
We already have a Spanish-Portuguese filter, add the domain to the existing rule in that filter:
```
##.ads
```
Also need to remove the cookie popup and share-button leftover, please add the domain to the existing rules:

```
##.cookie-consent
##.share-bar
```

File: SpanishFilter/sections/specific.txt [hunk 1]
```diff
@@ -16,10 +16,12 @@ $script,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|staticall
...
 ! ##body .banner
 tvonline.fan,laprensagrafica.com,ahoragranada.com,correio24horas.com.br,cnnespanol.cnn.com,motociclismo.es,sub100.com.br,theenemy.com.br,manhwas.net,rpp.pe,ambito.com,economipedia.com,informador.mx,elinformador.com.co,cronista.com,rionegro.com.ar,diariodecuyo.com.ar,voydeviaje.com.ar,guarda14.com,viapais.com.ar,agazeta.com.br,eleconomista.com.mx,metropolitanafm.com.br,tycsports.com,elespectador.com,bnews.com.br,migalhas.com.br,losandes.com.ar,lavoz.com.ar,mundo.sputniknews.com,omelete.com.br,lapoliticaonline.*,valenciaplaza.com,veneapp.com,dalealbo.cl,globo.com,defesanet.com.br,mibrujula.com,lyon.kim,26noticias.com.ar,auniao.com,clickpb.com.br,emagister.com,lanacion.com.ar,parana-online.com.br,r7.com,toxico-pc.com,uol.com.br##body .banner
 !
+amomeupet.org##.cookie-consent
```
Comment (Sergey-Lyapin) [inline SpanishFilter/sections/specific.txt:23]: Это в \AnnoyancesFilter\Cookies\sections\cookies_specific.txt к существующему правилу

File: SpanishFilter/sections/specific.txt [hunk 2]
```diff
@@ -16,10 +16,12 @@ $script,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|staticall
...
+amomeupet.org##.share-bar
```
Comment (Sergey-Lyapin) [inline SpanishFilter/sections/specific.txt:24]: \AdguardFilters\SocialFilter\sections\specific.txt

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt [hunk 3]
```diff
@@ -75,6 +75,7 @@ mentalfloss.com,90min.com,fansided.com,90min.de,12thmanrising.com,1428elm.com,8p
...
 !
+amomeupet.org##.share-bar
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:78]: please add domain to the existing rule:
```
rpgfan.com,techrepublic.com,9gag.com,aviasales.ru,huffingtonpost.jp,foxsports.com.mx,elfinanciero.com.mx,kursiv.media,talk.tw,designboom.com,elle.fr,mangacross.jp,manga-meets.jp,peopletalk.ru,foodallergy.org,sharecuts.cn,wtop.com,thegrammarexchange.infopop.cc,open.online,m.tecmundo.com.br,redbull.com,coindesk.com,playbuzz.com,huffingtonpost.co.uk,m.huffpost.com,digital-photography-school.com,tomshardware.com,tomshardware.co.uk##.share-bar
```

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 4]
```diff
@@ -266,6 +266,7 @@ codesandbox.io,damota.me,pracazarohom.*,seduo.*#%#//scriptlet('remove-class', 'f
 !
 ! NOTE: Regular rules
 !
+amomeupet.org##.cookie-consent
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:269]: Please add domain to the existing rule:
```
egrp365.ru,sqlservercentral.com,saneibd.com,edukasicampus.net,godiva.co.jp,0patch.com,pvpoke-re.com,radioua.net,greenday.com,linuc.org,clicsante.ca,israelnationalnews.com,serpstat.com,pandavpnpro.com,freeyourmusic.com,liica.co.jp,x5.ru,adshnk.com,parkernathan.com,route-inn.co.jp,pastelink.net,shuchi.php.co.jp,dictum.com,tamtam.chat,ynjn.jp,881903.com,irisohyama.co.jp,olyrix.com,rutube.ru,asm.org,socialcomics.com.br,margarethowell.jp,ascoltareradio.com,wego.jp,storececotec.it,geeksforgeeks.org,dauc.cz,heidisql.com,sueddeutsche.de,plex.tv,tmohentai.com,gorsovet.com.ua##.cookie-consent
```

**Review by Sergey-Lyapin — APPROVED** [review]

**Review by piquark6046 — APPROVED** [review]

**Review by BlazDT — APPROVED** [review]

**Pull request merged**